app/src/main/java/app/revanced/manager/ui/viewmodel/SelectedAppInfoViewModel.kt

```diff
@@ -151,8 +151,6 @@ class SelectedAppInfoViewModel(
 
         // Try to get the previous selection if customization is enabled.
         viewModelScope.launch {
-            if (!prefs.disableSelectionWarning.get()) return@launch
```

Comment:
Removing the early return will make manager use old selections, which shouldn't happen unless patch customization is enabled.